app/controllers/patients_controller.rb

```diff
@@ -249,53 +249,170 @@ def update
     render json: patient
   end
 
-  def update_hoh
+  # Moves a record into a household
+  def move_to_household
     new_hoh_id = params.permit(:new_hoh_id)[:new_hoh_id]&.to_i
-    current_patient_id = params.permit(:id)[:id]
-    household_ids = params[:household_ids] || []
+    current_patient_id = params.permit(:id)[:id]&.to_i
+
     current_patient = current_user.get_patient(current_patient_id)
-    old_hoh = current_user.get_patient(current_patient.responder_id)
     new_hoh = current_user.get_patient(new_hoh_id)
+    current_user_patients = current_user.patients
+
+    # ----- Error Checking -----
+
+    # Check to make sure selected HoH record exists.
+    unless Patient.where(purged: false).exists?(new_hoh_id)
```

Comment:
is there a case where we would allow a user to set the HoH of a patient as a non viewable patient of the user? just wondering if we would be able to do this instead:

`unless current_user_patients.exists?(new_hoh_id)`

Reply:
In theory, no that shouldn't be possible. Updated here: 131e345